### PR TITLE
Sp4 acctverif fe

### PR DESCRIPTION
**Pull request by parkaransingh**
Issue #63

As a user, I want to be able to be able to use a front-end to verify my account, so that I can verify even with less technical proficiency.

collaborators: Parkaranjot Singh And Madhura gangal

### CODE REVIEW

File: Civility Optics/Views/Onboarding/RegisterView.swift [hunk 1]
```diff
@@ -300,17 +300,18 @@ struct RegisterView: View {
...
+//                    TabView {
+//                        SearchView(model: .init(), email: email).tabItem {
+//                            Label("Search Places", systemImage: "magnifyingglass")
+//                        }
+//                        profile(model: UserProfileModel(email: self.email)).tabItem {
+//                            Label("Profile", systemImage: "person.circle.fill")
+//                        }
+//
+//                    }
+
```
Comment (madhura112) [inline Civility Optics/Views/Onboarding/RegisterView.swift:314]: we probably can get rid of this

**Review by madhura112 — APPROVED** [review]
Looks good!

File: Civility Optics/Views/Onboarding/RegisterView.swift [hunk 2]
```diff
@@ -300,17 +300,18 @@ struct RegisterView: View {
...
-                        SearchView(model: .init(), email: email).tabItem {
-                            Label("Search Places", systemImage: "magnifyingglass")
-                        }
-                        profile(model: UserProfileModel(email: self.email)).tabItem {
-                            Label("Profile", systemImage: "person.circle.fill")
-                        }
-                        
-                    }
-                    
+//                    TabView {
```
Comment (Spellbreeze) [inline Civility Optics/Views/Onboarding/RegisterView.swift:305]: Delete commented-out code.

File: Civility Optics/Views/Onboarding/VerifyView.swift [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            }.onChange(of: model.verified) { newValue in
+                didVerify = newValue
+            }
+        }
+    }
+}
+
+struct VerifyView_Previews: PreviewProvider {
+    static var previews: some View {
+        VerifyView(email: "Blah")
```
Comment (Spellbreeze) [inline Civility Optics/Views/Onboarding/VerifyView.swift:65]: What is this?

**Review by Spellbreeze — APPROVED** [review]

File: Civility Optics/Views/Onboarding/VerifyView.swift (hunk 3 above)
Comment (parkaransingh) [reply Civility Optics/Views/Onboarding/VerifyView.swift:65]: Preview struct. Doesnt affect the logic of the app.

File: Civility Optics/Views/Onboarding/RegisterView.swift (hunk 2 above)
Comment (parkaransingh) [reply Civility Optics/Views/Onboarding/RegisterView.swift:305]: ok

**Pull request merged**